angr/misc/reprs.py

```diff
@@ -0,0 +1,14 @@
+
+from archinfo.arch_soot import SootAddressDescriptor, SootMethodDescriptor
+
+
+def repr_addr(addr, x=True):
+    if type(addr) in (int, long):
+        if x:
```

Comment:
This `if` check seems redundant to me.

Reply:
This file is an old file that slipped through during the poll request. It is not used anymore. Removed now.